### ISSUE TITLE
[0.4] admin/server/api/download.js is orphaned

### ISSUE THREAD

**Issue by geloescht** [keystonejs/keystone#2406]
This appears to be superseded by admin/server/api/list/download.js (ec4e2ad7175affcf67af9365). The old file is referenced nowhere in the whole repository now.
Removal of this file will partially solve #1150.

**Comment by JedWatson** [comment]
Good point @geloescht. I left it in place because the new `admin/server/api/list/download.js` implementation doesn't completely handle the features implemented in `admin/server/api/download.js` so it was there as reference... I'll finish that and clean it out today.
